guides/tutorial/part-2/ember-data.md

```diff
@@ -1,22 +1,20 @@
-<!-- Heads up! This is a generated file, do not edit directly. You can find the source at https://github.com/ember-learn/super-rentals-tutorial/blob/master/src/markdown/tutorial/part-2/11-ember-data.md -->
+Dans ce chapitre, nous allons nous atteler à supprimer du code dupliqué dans nos gestionnaires de routes (_route handlers_), en utilisant Ember Data pour gérer nos données. Visuellement, le résultat final sera exactement comme avant&nbsp;:
```

Comment:
Not necessarily requesting a change, eventually but not mandatory, but do we want to stay consistent when using "EmberData" or "Ember Data"?

The Ember.js team uses the format `EmberData`:
https://github.com/emberjs/data/

Reply:
Oh, that's very interesting. There is a PR in the English guides that [replaces all the references to "Ember Data" with "EmberData"](https://github.com/ember-learn/guides-source/pull/1896/files), and a while ago I did a "catchup English" that takes this into account. But the tutorial part was not modified in this PR because the tutorial part is generated from elsewhere on the English side.